### PR TITLE
Pmm 9526 migrate enzyme to rtl

### PR DESCRIPTION
**Pull request by solovevayaroslavna**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
https://jira.percona.com/browse/PMM-9526

### CODE REVIEW

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: public/app/percona/backup/components/AddBackupModal/RetryModeSelector/RetryModeSelector.test.tsx [hunk 1]
```diff
@@ -1,37 +1,38 @@
...
         <RetryModeSelector retryMode={RetryMode.AUTO} />
       </FormWrapper>
     );
-    expect(wrapper.find(RadioButtonGroupField)).toHaveLength(1);
-    expect(wrapper.find(NumberInputField)).toHaveLength(2);
-    expect(wrapper.find(NumberInputField).at(0).prop('disabled')).toBeFalsy();
+    expect(screen.getAllByTestId('retryMode-radio-state')).toHaveLength(1);
+    expect(screen.getAllByTestId('retryTimes-field-container')).toHaveLength(1);
+    expect(screen.getAllByTestId('retryInterval-field-container')).toHaveLength(1);
+    expect(screen.getByTestId('retryTimes-number-input')).toHaveProperty('disabled', false);
```
[the remark this answers is not in the document]
Comment (solovevayaroslavna) [reply public/app/percona/backup/components/AddBackupModal/RetryModeSelector/RetryModeSelector.test.tsx:17]: yes, thank you for the remark! I did not immediately find out about this method and did not rewrite it everywhere, I will correct it!

**Comment by fabio-silva** [comment]
@solovevayaroslavna I can still see change on yarn.lock. Have we reverted it?

**Comment by solovevayaroslavna** [comment]
@fabio-silva I accidentally commented it out again during edits with disabled, so I brought it back to the main state again.

**Comment by solovevayaroslavna** [comment]
@fabio-silva If you look at the general PR, then it has already been fixed there

**Review by fabio-silva — APPROVED** [review]

**Pull request merged**